resources/assets/tconstruct/lang/sv_se.lang

```diff
@@ -0,0 +1,1041 @@
+
+##################
+#   UI Strings   #
+##################
+# general strings used in GUIs, tooltips and other display stuff
+
+ui.mininglevel.stone=Sten
+ui.mininglevel.iron=Järn
+#ui.mininglevel.redstone=Redstone # replaced with diamond
+ui.mininglevel.diamond=Diamant
+ui.mininglevel.obsidian=Obsidian
+ui.mininglevel.cobalt=Kobolt
+
+# Death messages
+death.attack.smeltery=%s försökte smälta spelare
+death.attack.splinter=%s fick en flisa i sig som måste ha varit enorm
+death.attack.bleed=%1$s blödde ihjäl från %2$ss attack
+death.attack.bleed.item=%1$s blödde ihjäl från %2$ss attack med %3$s
+
+message.mending_moss.not_enough_levels=Lagningsmossan kräver minst %s nivåer
+tconstruct.channel.connected.in=Ändrade sidan till att flöda inåt
+tconstruct.channel.connected.out=Ändrade sidan till att flöda utåt
+tconstruct.channel.connected.none=Flöde åt sidan tillåts inte
+tconstruct.channel.connected_down.allow=Nedåtriktat flöde tillåts
+tconstruct.channel.connected_down.disallow=Nedåtriktat flöde tillåts inte
+
+######################
+# GUI Error messages #
+######################
+
+gui.error=FEL
+gui.warning=VARNING
+# GUI Errors from the different Tinker Tables
+gui.error.invalid_pattern=Mönstret innehåller inte en giltig verktygsdel!
+gui.error.useless_tool_part=Denna del kan inte användas för att tillverka ett verktyg!\nAntingen saknar materialet %s lite information eller så använder inget verktyg %s i din tillverkning.
+gui.error.wrong_material_part=En eller flera föremål har ett material som inte stöds för detta verktyg.
+gui.error.not_enough_durability=Inte tillräckligt mycket hållbarhet för att ersätta delar!\nBehöver %d mer hållbarhet.
+gui.error.craftevent.tool.default=En annan mod förhindrar att detta föremål tillverkas.
+gui.error.craftevent.modifier.default=En annan mod förhindrar att detta verktyg modifieras med dessa föremål.
+gui.error.craftevent.toolpart.default=En annan mod förhindrar att denna verktygsdel tillverkas.
+gui.error.craftevent.replace.default=En annan mod förhindrar att denna verktygsdel ersätts.
+# GUI Errors from modifiers
+gui.error.not_enough_modifiers=Inte tillräckligt många modifierare. (%d behövs)
+gui.error.single_modifier=%s kan endast tillämpas en gång.
+gui.error.max_level_modifier=Den maximala nivån för %s är redan nådd.
+gui.error.no_modifier_for_item=Ingen modifierare hittades för %s.
+gui.error.already_has_extratrait=Verktyget har redan en prägling och kan bara ha en.
+gui.error.incompatible_modifiers=Modifierarna %s och %s kan inte tillämpas tillsammans
+gui.error.incompatible_trait=Modifieraren %s kan inte användas tillsammans med attributen %s
+gui.error.incompatible_enchantments=Modifieraren %s kan inte kombineras med förtrollningen %s
+gui.error.no_rename=Detta verktyg kan inte byta namn
+
+################
+#     GUIs     #
+################
+
+# General things used in multiple GUIs
+gui.general.hover=Lägg musen över uppslagen för mer information
+gui.general.temperature.celsius=%s °c
```

Comment:
I'd remove the space here, it looks a bit awkward in game and is not in the english version. I also switched this to a capital C in the english one recently, so just make sure this is correct for your language's symbol for Celsius
![2019-03-20_22 45 41](https://user-images.githubusercontent.com/12155288/54732565-8282d000-4b62-11e9-9a11-74323e9e6120.png)

Reply:
Swedish spelling rules says that there must be a space before a grade sign in temperature related situations, but it looks kinda tight and will most likely overflow if it reaches four digits, so I'll remove it.
C and F should be uppercase, but I must have made a blunder there. Thanks for being observant!